src/tools/pontoon/resources.md

```diff
@@ -41,9 +49,13 @@ Above the entry, a diff view between the current source string and the source st
 
 To the right of the entry, the number in green shows the percent match between the machinery string and the source string. The higher the percentage, the more likely the machinery is to be useful. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory source.
```

Comment:
The two paragraphs above no longer match the screenshots. The first one probably needs to be removed, because Machinery search has been replaced with Concordance search, which is covered below.